lib/config.js

```diff
@@ -130,6 +134,10 @@ const conf = convict({
     port: {
       default: '3306',
       env: 'MYSQL_PORT'
+    },
+    connectionLimit: {
```

Comment:
This whole `mysql` config object is passed to `mysql.createPool`, so it will be one of the options to the constructor.